dll/win32/comctl32/propsheet.c

```diff
@@ -3580,6 +3580,26 @@ PROPSHEET_DialogProc(HWND hwnd, UINT uMsg, WPARAM wParam, LPARAM lParam)
         }
         SetFocus(GetDlgItem(hwnd, IDOK));
       }
+#ifdef __REACTOS__
+      { /* try to fit it into the desktop */
+          RECT rcWork;
+          RECT rcDlg;
+          int dx, dy;
+
+          if (GetWindowRect(hwnd, &rcDlg) && SystemParametersInfo(SPI_GETWORKAREA, 0, &rcWork, 0))
+          {
+              dx = rcDlg.right - rcWork.right;
+              dy = rcDlg.bottom - rcWork.bottom;
+
+              if (dx > 0)
+                  rcDlg.left -= dx;
+              if (dy > 0)
+                  rcDlg.top -= dy;
+
+              SetWindowPos(hwnd, HWND_TOPMOST, rcDlg.left, rcDlg.top, 0, 0, SWP_NOSIZE);
```

Comment:
Sorry, I wasn't clear: you'll also want most of the other SWP_NOXX flags.
Looking at https://git.reactos.org/?p=reactos.git;a=blob;f=win32ss/user/ntuser/winpos.c;hb=782ff23261613bb745d553d95addbe1befbd4c8f#l3371, that would be:
`SWP_NOZORDER | SWP_NOACTIVATE | SWP_NOREDRAW | SWP_NOSIZE`